lib/cucumber/formatter/console.rb

```diff
@@ -80,51 +78,23 @@ def print_element_messages(element_messages, status, kind)
         end
       end
 
-      def print_stats(features, options)
```

Comment:
Fair point Bjorn. I'd like to get rid of that module altogether really, or at least slim down its interface a lot. Using composition (e.g. `ConsoleIssues`) is the way forward.